### PR TITLE
Add correct url for mainnet staging tx service; remove relay service …

### PR DESCRIPTION
**Pull request by elgatovital**
Add correct url for mainnet staging tx service

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -79,7 +79,6 @@
     <string name="endpoints">ENDPOINTS</string>
     <string name="rpc_endpoint">RPC endpoint</string>
     <string name="tx_service">Transaction service</string>
-    <string name="relay_service">Relay service</string>
```
Comment (biafra23) [inline app/src/main/res/values/strings.xml:82]: 👍

File: data/build.gradle [hunk 2]
```diff
@@ -19,37 +19,31 @@ android {
...
         rinkeby {
             buildConfigField javaTypes.STRING, "TRANSACTION_SERVICE_URL", asString(getKey("TRANSACTION_SERVICE_URL", "https://safe-transaction.rinkeby.gnosis.io/api/"))
-            buildConfigField javaTypes.STRING, "RELAY_SERVICE_URL", asString(getKey("RELAY_SERVICE_URL", "https://safe-relay.rinkeby.gnosis.io/api/"))
             buildConfigField javaTypes.STRING, "CLIENT_GATEWAY_URL", asString(getKey("CLIENT_GATEWAY_URL", "https://safe-client.rinkeby.gnosis.io/"))
         }
 
         mainnet_staging {
-            buildConfigField javaTypes.STRING, "TRANSACTION_SERVICE_URL", asString(getKey("TRANSACTION_SERVICE_URL", "https://safe-transaction.staging.gnosisdev.com/api/"))
-            buildConfigField javaTypes.STRING, "RELAY_SERVICE_URL", asString(getKey("RELAY_SERVICE_URL", "https://safe-relay.staging.gnosisdev.com/api/"))
+            buildConfigField javaTypes.STRING, "TRANSACTION_SERVICE_URL", asString(getKey("TRANSACTION_SERVICE_URL", "safe-transaction.mainnet.staging.gnosisdev.com/api/ "))
```
Comment (biafra23) [inline data/build.gradle:41]: We are still using this service for notifications, right?

**Review by biafra23 — APPROVED** [review]
LGTM 👍

File: data/build.gradle (hunk 2 above)
Comment (elgatovital) [reply data/build.gradle:41]: yes

**Pull request merged**